test/assets/ChargeHistoryTestConstants.scala

```diff
@@ -116,15 +116,15 @@ object ChargeHistoryTestConstants {
     idValue = "XAIT000000000000",
     regimeType = "ITSA",
     chargeHistoryDetails = Some(List(
-      ChargeHistoryDetailModel("  2019 ", "123456789", LocalDate.parse("2020-01-29"), "Balancing Charge", 12345678912.12, "2020-02-24", "amended return")
+      ChargeHistoryDetailModel("  2019 ", "123456789", LocalDate.parse("2020-01-29"), "Balancing Charge", 12345678912.12, "2020-02-24", "amended return", None)
```

Comment:
All the added instances of poaAdjustmentReason are None in this test data. Should we be testing with any thing else? Should we be showing that when we have data it is parsed and passed on correctly?